katas/content/multi_qubit_gates/two_qubit_gate_1/Verification.qs

```diff
@@ -0,0 +1,25 @@
+namespace Kata.Verification {
+    open Microsoft.Quantum.Katas;
+
+    operation  TwoQubitGate1 (qs : Qubit[]) : Unit is Adj + Ctl {
+        CNOT(qs[0], qs[1]);
+    }
+
+    operation CheckSolution() : Bool {
+        let solution = Kata. TwoQubitGate1;
+        let reference =  TwoQubitGate1;
+        let isCorrect = CheckOperationsEquivalence(solution, reference, 2);
```

Comment:
In this test, we don't need to check the equivalence of operations as unitaries, since the tasks asks us to prepare a state given another specific state. (For example, we can apply Z gate to the 2nd qubit before the CNOT, it would be a different unitary but a correct solution too.)

The test we need here is very similar to what I'm doing in https://github.com/microsoft/qsharp/blob/mariia/qec_shor/katas/content/qec_shor/bitflip_encode/Verification.qs using the operation `CheckOperationsEquivalenceOnInitialStateStrict` you created in your previous PR: compare the effect of the solution and the reference on several specific input states with different parameters. (I'm also refactoring the code to move it to the KatasLibrary.qs as part of that change - you can see the complete diff in https://github.com/microsoft/qsharp/compare/mariia/qec_shor?expand=1).

My change won't be ready for a few more days, though. For your PR, the easiest way to do is copy-paste CheckOperationsEquivalenceOnInitialStateStrict to this Verification.qs file and copy the test from my branch; I'll include that in the refactoring once it's merged and I'll be creating my own pull request.